### ISSUE TITLE
Need tuning suggestions

### ISSUE THREAD

**Issue by rockdrigoma** [lucidrains/stylegan2-pytorch#121]
Hi all,
I trying to train a model with ~1K and ~5K images.
So far model is collapsing, or this is what I understand from output.

I tried first with 1K , following parameters and 150K epochs:
`stylegan2_pytorch --aug_prob 0.25 --attn_layers [1,2]`

And got:
![image](https://user-images.githubusercontent.com/5482256/91085710-792d4b00-e613-11ea-9acb-bb3b6ce0b4b6.png)

Then I tried with ~5K, following parameters and 150K epochs:
`stylegan2_pytorch --aug_prob 0.25 --attn_layers [1,2]`

And got:
![image](https://user-images.githubusercontent.com/5482256/91086002-e0e39600-e613-11ea-897f-35b44a973a23.png)

What do you suggest to do to improve results?
There are a lot of parameters to try and by reading other threads _cl_reg_ and _trunc_psi_ are suggested but by using them, the images start to get worse. Right now I'm trying to reduce _batch_size_ to 3 and increase _network_capacity_ to 32

    data = './data' 
    results_dir = './results'
    models_dir = './models'
    name = 'default'
    new = False
    load_from = -1
    image_size = 128
    network_capacity = 16
    transparent = False
    batch_size = 5 
    gradient_accumulate_every = 6
    num_train_steps = 150000
    learning_rate = 2e-4
    lr_mlp = 0.1
    ttur_mult = 1.5
    num_workers =  None
    save_every = 1000
    generate = False
    generate_interpolation = False
    save_frames = False
    num_image_tiles = 8
    trunc_psi = 0.75
    fp16 = False
    cl_reg = False
    fq_layers = []
    fq_dict_size = 256
    attn_layers = []
    no_const = False
    aug_prob = 0. 
    dataset_aug_prob = 0.

**Comment by lucidrains** [comment]
Please try https://github.com/lucidrains/unet-stylegan2 ! And yes, up the network capacity and make sure the batch size x gradient accumulate is at least 32

**Comment by lucidrains** [comment]
Who is that??

**Comment by rockdrigoma** [comment]
> Who is that??

You mean the person being generated? No one,  I crawled images of Latin American models from online clothe stores by hand and others by a crawler.

**Comment by rockdrigoma** [comment]
> Who is that??

Should I keep the attention layers and aug_prob?
What's the difference between aug_prob and dataset_aug_prob?

**Comment by lucidrains** [comment]
yup, keep attention and aug prob! dataset aug prob augments the images immediately when they are fetched from the file system, and not when it is fed to the discriminator. the former is not differentiable while the latter is

**Comment by AlexTS1980** [comment]
I also trained on dataset ~1K using tricks like attention in different modules, `--aug-prob .25 --dataset-aug-prob .5 --trunc-psi 1` ( last one varying from 0.5 to 1, the other two from 0 to 0.5), but it keeps converging like this (logo generator)

![generated-08-12-2020_23-04-44](https://user-images.githubusercontent.com/8479872/91489563-ed345100-e8a8-11ea-9469-7cacba7ecf53.jpg)

**Comment by rockdrigoma** [comment]
> --trunc-psi

try by removing the --trunc-psi parameter
